concourse/steps/scan_container_images.py

```diff
@@ -256,32 +299,24 @@ def protecode_scan(
     processing_mode,
     parallel_jobs: int,
     cve_threshold,
-    include_image_references,
-    exclude_image_references,
+    image_reference_filter,
 ):
     # print configuration
     table_data = (
         ('Protecode target group id', str(protecode_group_id)),
         ('Protecode group URL', protecode_group_url),
         ('Protecode reference group IDs', reference_protecode_group_ids),
-        ('Image Filter (include)', include_image_references),
-        ('Image Filter (exclude)', exclude_image_references),
```

Comment:
please mv-back those two lines and append the other four filters

Reply:
I've split the protecode function in two, one function that displays this info table and one that mostly passes on the arguments to the proper scanning function now (as I did not want to move this logic _back_ to mako)

While I've done as suggested, I don't think the filter config belongs into the protecode part, as it is active for both protecode and clamav scans.

Reply:
that's true. some further refactoring would make sense..